apps/site/src/content/blog/state-of-the-budgie-2024.mdx

```diff
@@ -0,0 +1,273 @@
+
+**Applets**
+
+Some applets need to be updated to reach complete or some degree of feature parity with their usage under the "legacy" X11 environment.
+
+- Keyboard Layout applet: This currently uses IBus and `Gnome.Languages` and needs to be updated so keyboard layout changing via the applet works.
+- Night Light applet: This needs to be updated to leverage the [wlr gamma control protocol](https://wayland.app/protocols/wlr-gamma-control-unstable-v1). It will most likely do this by effectively being an indicator similar to gammastep-indicator, leveraging [gammastep](https://gitlab.com/chinstrap/gammastep) for starting / stopping gammastep and setting a specific temperature. Budgie Daemon v2 will eventually handle gamma setting for us.
+- Tasklist applet: Not to be confused with IconTasklist, this "legacy" tasklist provides a horizontal list of running apps, ungrouped, similar to `Wnck.Tasklist`. Evan has started rewriting this however some more work needs to be done on this front.
+- Workspaces applet: Lots of xid usage still which isn't available for obvious reasons under Wayland.
+
+**Display Configuration**
+
+I need to complete my implementation of the Display Configuration Batch system in budgie-daemon v2, it needs to go through a bunch of testing, and we need to update Budgie Control Center to use it. Lots of work to be done here still.
+
+**Misc**
+
+MenuManager needs to be cleaned up / deprecated. This provides a right-click menu to show options for Budgie Desktop Settings and System Settings. Budgie Desktop View has its own implementation, however MenuManager handles it when desktop icons are turned off. This can be handled simply by updating our labwc rc.xml template.
+
+#### Shipping
+
+We are currently intent on shipping Budgie 10.10 within Q1 2025 (Jan-Mar), however as always we won't ship it unless we consider it ready (at least as ready as we can test). Budgie 10.10 will ship with the recommendation and default of using labwc, providing us more time to bake an excellent compositor that we can swap out at any time across Budgie 10 or 11!
+
+It will be too late for Fedora 42 and Ubuntu 25.04, so the current plan is:
+
+1. Ship it in Fedora rawhide once it is branched for Fedora 43 as well as have a COPR for Fedora 42. Budgie 10.10(.x) will ship in Fedora 43, with a self-contained change proposal planned to be submitted after the 10.10 release. While it won't be available immediately with Budgie 10.10, the plan is to also offer an ISO with the wayland session set up out-of-the-box too.
+2. Ubuntu Budgie folks will provide an on-ramp in the form of a PPA for interested testers with the plan on shipping for Ubuntu 25.10.
+3. Solus will have either a testing repo or make available eopkg downloads for testing. Since it is a rolling release, they have the luxury of being able to ship it whenever, so they will do it after would-be testers have put it through its paces and we have worked out any issues that'd be considered blockers.
```

Comment:
```suggestion
3. Solus will have either a testing repo, or otherwise make available eopkg downloads for testing. Since it is a rolling release, they have the luxury of being able to ship it whenever, so they will do it after would-be testers have put it through its paces and we have worked out any issues that'd be considered blockers.
```